pithos/pithos.py

```diff
@@ -161,18 +161,20 @@ def __init__(self, app, test_mode):
         self.set_proxy(reconnect=False)
         self.set_audio_quality()
 
+        keyring_error = unlock_keyring()
```

Comment:
Instead of having it return an error just let the exception propagate and handle it here?